autostart/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.0.2 FATAL_ERROR)
+
+file(GLOB DESKTOP_FILES *.desktop)
+
+
+add_custom_target(autostart_desktop_files ALL DEPENDS ${DESKTOP_FILES})
+#************************************************
+
+install(FILES
+    ${DESKTOP_FILES}
+    DESTINATION "${LXQT_ETC_XDG_DIR}/autostart"
+    COMPONENT Runtime
```

Comment:
Ok.
I splitted files into two componets:
* Development - for files only needed to build the package
* Runtime - for files needed at runtime, another way of saying everything else. 

What component do you have in mind ?